src/apscheduler/_schedulers/async_.py

```diff
@@ -30,7 +30,13 @@
 from .. import JobAdded, SerializationError, TaskLookupError
 from .._context import current_async_scheduler, current_job
 from .._converters import as_enum, as_timedelta
-from .._enums import CoalescePolicy, ConflictPolicy, JobOutcome, RunState, SchedulerRole
+from .._enums import (
+    CoalescePolicy,
+    ConflictPolicy,
+    JobOutcome,
+    RunState,
+    SchedulerRole,
+)
```

Comment:
I notice your PR changes the formatting of the imports in a couple of places, seemingly without a need. Just FYI.

Reply:
Addressed in 695efe6